### PR TITLE
style 9FEB2021

### PR DESCRIPTION
**Pull request by shssf**

### CODE REVIEW

**Review by samir-nasibli — APPROVED** [review]

File: dpnp/dpnp_algo/dpnp_algo.pyx [hunk 1]
```diff
@@ -68,8 +68,8 @@ include "dpnp_algo_statistics.pyx"
 include "dpnp_algo_trigonometric.pyx"
 
 
-ctypedef void(*fptr_dpnp_arange_t)(size_t, size_t, void * , size_t)
-ctypedef void(*fptr_dpnp_initval_t)(void * , void * , size_t)
+ctypedef void(*fptr_dpnp_arange_t)(size_t, size_t, void *, size_t)
+ctypedef void(*fptr_dpnp_initval_t)(void *, void * , size_t)
```
Comment (Alexander-Makaryev) [inline dpnp/dpnp_algo/dpnp_algo.pyx:72]: Pay attention to spaces at ` void * `. Here our code style tool can't decide should we have a space after star or not, and it happens in the same line and same definition.

File: dpnp/dpnp_algo/dpnp_algo.pyx [hunk 2]
```diff
@@ -84,7 +84,7 @@ cpdef dparray dpnp_arange(start, stop, step, dtype):
     cdef DPNPFuncType param1_type = dpnp_dtype_to_DPNPFuncType(dtype)
     cdef DPNPFuncData kernel_data = get_dpnp_function_ptr(DPNP_FN_ARANGE, param1_type, param1_type)
 
-    result_type = dpnp_DPNPFuncType_to_dtype(< size_t > kernel_data.return_type)
+    result_type = dpnp_DPNPFuncType_to_dtype( < size_t > kernel_data.return_type)
```
Comment (Alexander-Makaryev) [inline dpnp/dpnp_algo/dpnp_algo.pyx:87]: Code style tool:
- We need a space before `<` !

File: dpnp/dpnp_algo/dpnp_algo.pyx [hunk 3]
```diff
@@ -138,7 +138,7 @@ cpdef dparray dpnp_init_val(shape, dtype, value):
 
     cdef DPNPFuncData kernel_data = get_dpnp_function_ptr(DPNP_FN_INITVAL, param1_type, param1_type)
 
-    result_type = dpnp_DPNPFuncType_to_dtype( < size_t > kernel_data.return_type)
+    result_type = dpnp_DPNPFuncType_to_dtype(< size_t > kernel_data.return_type)
```
Comment (Alexander-Makaryev) [inline dpnp/dpnp_algo/dpnp_algo.pyx:141]: Code style tool (continue):
-Oh no! We don't need a space before < !
)

**Review by Alexander-Makaryev — APPROVED** [review]

File: dpnp/dpnp_algo/dpnp_algo.pyx (hunk 1 above)
Comment (samir-nasibli) [reply dpnp/dpnp_algo/dpnp_algo.pyx:72]: Seems like codestyle tool doesn't work correct for Cython files.

Comment (shssf) [reply dpnp/dpnp_algo/dpnp_algo.pyx:72]: This is one of the thing we need to improve

**Pull request merged**